### PR TITLE
blog: Explain MQTT

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
With this post it gives a brief overview of MQTT and explains how to configure a broker.

Closes https://github.com/flowforge/content/issues/120

### CODE REVIEW

File: src/blog/2023/05/connect-to-hivemq-in-flowforge.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: MQTT and its Role in IoT and Industrial IoT
+subtitle: A Guide to Connecting HiveMQ with Node-RED
+description: Node-RED has great support for MQTT, in this guide we'll explain why that's fundamental for IoT and how to connect to a broker
+date: 2023-06-10
+authors: ["zeger-jan-van-de-weg"]
+image: # /blog/2023/02/images/tile-TODO.jpg
```
Comment (ZJvandeWeg) [inline src/blog/2023/05/connect-to-hivemq-in-flowforge.md:7]: Pending: https://github.com/flowforge/content/issues/120

File: src/blog/2023/05/images/node-red-config-mqtt-server.png
Comment (iskerrett) [inline src/blog/2023/05/images/node-red-config-mqtt-server.png:1]: The server url looks like a HiveMQ Cloud url.  Are you using HiveMQ CLoud or their Public broker. Their public broker doesn't use TLS.

File: src/blog/2023/05/connect-to-hivemq-in-flowforge.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+1. **Reliability**: MQTT incorporates a Quality of Service (QoS) mechanism that ensures message delivery, making it ideal for applications that require reliable data transmission.
+1. **Asynchronous Communication**: MQTT enables asynchronous communication, allowing devices to operate independently, reducing latency and increasing system responsiveness.
+
+## Sending messages to a MQTT broker with Node-RED
+
+As broker we'll use the [public MQTT broker](https://www.hivemq.com/public-mqtt-broker/) provided by HiveMQ. We'll use FlowForge to host and manage the Node-RED instance for us. After opening the editor
+the MQTT nodes are pre-installed as these are "Core Nodes" in Node-RED.
+
+Drag in the "Inject" node, as well as the "MQTT out" node and connect the Injects output to the MQTT
+out node. Now it's time to configure the MQTT node to connect to the broker. On HiveMQ you should generate both a username and password, afterwards copy and paste those to the Node-RED MQTT Server
```
[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply src/blog/2023/05/connect-to-hivemq-in-flowforge.md:31]: I used the "Free tier" and had to use a username and password?

File: src/blog/2023/05/images/mqtt-in-config-node-red.png
[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply src/blog/2023/05/images/mqtt-in-config-node-red.png:1]: Its the default set by Node-RED, and didn't want to explain the different. Could be content for the follow up though indeed

File: src/blog/2023/05/images/node-red-config-mqtt-server.png
Comment (ZJvandeWeg) [reply src/blog/2023/05/images/node-red-config-mqtt-server.png:1]: I'm using the free tier it seems, will update the wording

File: src/blog/2023/05/connect-to-hivemq-in-flowforge.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (iskerrett) [reply src/blog/2023/05/connect-to-hivemq-in-flowforge.md:32]: Ok, you are using HiveMQ Cloud, not their public broker that you linked to.

File: src/blog/2023/05/connect-to-hivemq-in-flowforge.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+![Configure the MQTT broker in Node-RED](./images/node-red-config-mqtt-server.png "Configuring a MQTT broker in Node-RED")
+
+After saving the config, and deploying the changes, the flow should tell display a green
+status bubble under the MQTT node which tells you it's connected and configured properly.
+
+![Connected MQTT node in Node-RED"](./images/connected-mqtt-node.png "Connected MQTT node in Node-RED")
+
+If we than set the Inject node to send a message every say 1 second, the timestamp will be
+send to the broker through MQTT!
```
[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply src/blog/2023/05/connect-to-hivemq-in-flowforge.md:43]: This is surprising, but true. 🤷 Weird language, English.

File: src/blog/2023/05/connect-to-hivemq-in-flowforge.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+config pane, along with the server details:
+
+![Configure the MQTT broker in Node-RED](./images/node-red-config-mqtt-server.png "Configuring a MQTT broker in Node-RED")
+
+After saving the config, and deploying the changes, the flow should tell display a green
+status bubble under the MQTT node which tells you it's connected and configured properly.
+
+![Connected MQTT node in Node-RED"](./images/connected-mqtt-node.png "Connected MQTT node in Node-RED")
+
+If we than set the Inject node to send a message every say 1 second, the timestamp will be
```
Comment (Yndira-FlowForge) [inline src/blog/2023/05/connect-to-hivemq-in-flowforge.md:42]:
```suggestion
If we then set the Inject node to send a message every say 1 second, the timestamp will be
```

**Comment by iskerrett** [comment]
fwiw, this post should be moved from the 05 folder to the 06 folder.

**Pull request merged**